### PR TITLE
Add new remote ID links to author pages

### LINKED ISSUE #5284: Update author "Links" rendering code to use /config/author
Currently, new identifiers added to /config/author won't appear in the "Links" section.

### Evidence / Screenshot (if possible)
image

### Relevant url?
eg https://openlibrary.org/authors/OL18319A/Mark_Twain

### Steps to Reproduce
1. Go to https://openlibrary.org/authors/OL18319A/Mark_Twain
2. Observe https://openlibrary.org/authors/OL18319A.json has ids for gutenberg and librivox

* Actual: Gutenberg/librivox not in "Links" section
* Expected: Gutenberg/librivox _are_ in "Links" section

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?** FF88
- **Operating system?** Win10
- **Environment (prod/dev/local)?** prod

### Stakeholders
@RayBB

### PR DESCRIPTION
**Pull request by jamesachamp**
Closes #5284

Adds LibriVox and Project Gutenberg links to author pages if a corresponding `remote_id` is present.

### Technical

In addition to the `/config/author.yml` configurations outlined in #5093, two new identifiers must be added:
```
-   label: Project Gutenberg
    name: project_gutenberg
    notes: ''
    url: https://www.gutenberg.org/ebooks/author/@@@
-   label: LibriVox
    name: librivox
    notes: ''
    url: https://librivox.org/author/@@@
```

### Testing

1. Add the correct LibriVox and Project Gutenberg IDs to an author's identifiers.
2. Confirm that the links are present in the "Links (outside Open Library)" section of the author page.
3. Confirm that the links navigate to the correct Project Gutenberg and LibriVox pages.

### Screenshot

remote_ids

### Stakeholders

@RayBB @cdrini

### CODE REVIEW

**Comment by cdrini** [comment]
Ah sorry, got the path flipped. Should be https://openlibrary.org/config/author ; Already added :)

File: openlibrary/templates/type/author/view.html [hunk 1]
```diff
@@ -179,6 +179,10 @@ <h3>$:_('Links <span class="gray small sansserif">(outside Open Library)')</span
                         <li>VIAF: <a itemprop="sameAs" href="https://viaf.org/viaf/$viaf">$viaf</a></li>
                     $if page.remote_ids.wikidata:
                         <li>Wikidata: <a itemprop="sameAs" href="https://tools.wmflabs.org/reasonator/?q=$page.remote_ids.wikidata">$page.remote_ids.wikidata</a></li>
+                    $if page.remote_ids.project_gutenberg:
```
Comment (cdrini) [inline openlibrary/templates/type/author/view.html:182]: Ah, https://openlibrary.org/config/author contains things like label/link format. We want to replace this with a call to `get_author_config()`, and then walk through all the `identifiers`.

Comment (cdrini) [reply openlibrary/templates/type/author/view.html:182]: Editions do something similar, so looking there might help?

Comment (jamesachamp) [reply openlibrary/templates/type/author/view.html:182]: That's a great approach!  Should the existing links also be replaced?  Also, could you provide a sample prod URL configuration if they are different from the configs in #5093 (`@@@` used in place of the ID)?

Comment (cdrini) [reply openlibrary/templates/type/author/view.html:182]: Can you see the page here: https://openlibrary.org/config/author ? It should be publicly visible. That's the prod config (Yep! @@@ is used). And yep! We should replace the old ones as well :0

Comment (jamesachamp) [reply openlibrary/templates/type/author/view.html:182]: > Can you see the page here: https://openlibrary.org/config/author ?

Yes, I can see it.  Not sure why I didn't check first before asking...  Thanks!

Comment (cdrini) [reply openlibrary/templates/type/author/view.html:182]: :D

Comment (jamesachamp) [reply openlibrary/templates/type/author/view.html]: Updated.  New screenshot:

remote_ids_updated

**Review by cdrini — APPROVED** [review]
Confirmed they show up on testing! http://testing.openlibrary.org/authors/OL215793A/Edward_Sylvester_Ellis

image

**Pull request merged**